src/components/Comment/Comment.tsx

```diff
@@ -151,46 +162,53 @@ export const Comment = ({
         </span>
       )}
       <div className={commentWrapper}>
-        <img
-          src={scopedComment.userProfile.avatar}
-          alt={scopedComment.userProfile.displayName}
-          className={cx(avatar(50), commentAvatar)}
-        />
+        <div className={avatarMargin}>
+          <Avatar
+            imageUrl={comment.userProfile.avatar}
+            displayName={comment.userProfile.displayName}
+            size={isReply ? "small" : "medium"}
+          />
+        </div>
 
         <div className={commentDetails}>
           <header className={headerStyles}>
             <Column>
               <Row>
                 <div className={commentProfileName(pillar)}>
-                  {scopedComment.userProfile.displayName}
+                  <a
+                    href={`https://profile.theguardian.com/user/${comment.userProfile.userId}`}
+                    className={linkStyles}
+                  >
+                    {comment.userProfile.displayName}
+                  </a>
                 </div>
                 <div className={timestampWrapperStyles}>
                   <Timestamp
-                    isoDateTime={scopedComment.isoDateTime}
-                    linkTo={`https://discussion.code.dev-theguardian.com/comment-permalink/${scopedComment.id}`}
+                    isoDateTime={comment.isoDateTime}
+                    linkTo={`https://discussion.code.dev-theguardian.com/comment-permalink/${comment.id}`}
```

Comment:
Not an issue for this PR but we'll need to be careful where we've got hardcoded refs to both production and code existing at the same time. We'll need to think about either sticking with prod or abstracting the baseUrl